functions/create_draft/blocks.ts

```diff
@@ -195,7 +191,7 @@ export const buildConfirmSendModal = (
         "text": {
           "type": "mrkdwn",
```

Comment:
```suggestion
          type: "mrkdwn",
```

Rambling here but I think it's interesting the formatter does not remove quotes from the attributes?